auth0/resource_auth0_custom_domain_verification.go

```diff
@@ -50,7 +50,11 @@ func createCustomDomainVerification(d *schema.ResourceData, m interface{}) error
 
 		d.SetId(customDomainVerification.GetID())
 
-		return resource.NonRetryableError(readCustomDomainVerification(d, m))
+		if err := readCustomDomainVerification(d, m); err != nil {
+			return resource.NonRetryableError(err)
+		}
+
+		return nil
```

Comment:
**Nit (non-blocking):** The best I can tell, `resource.NonRetryableError` will return nil in the restul of a nil error passed in. So I don't believe this additional conditional does anything.

Reply:
This is actually needed because of a behavior change in retryableError wrapper. From the docs:

>Clearer Handling of nil for helper/resource.NonRetryableError and helper/resource.RetryableError
Previously, passing a nil error to helper/resource.RetryableError and helper/resource.NonRetryableError could lead to subtle bugs and even crashes. It will now return an error. Providers should check that the error is not nil before calling helper/resource.NonRetryableError or helper/resource.RetryableError.

Reply:
Ah ok understood, thanks for the explanation 👍